edx_proctoring/backends/__init__.py

```diff
@@ -16,3 +16,6 @@ def get_backend_provider(exam=None):
         elif exam['backend']:
             backend_name = exam['backend']
     return apps.get_app_config('edx_proctoring').get_backend(name=backend_name)
+
+
+PING_FAILURE_PASSTHROUGH_TEMPLATE = 'edx_proctoring.{}_ping_failure_passthrough'
```

Comment:
How about moving this to constants? 

Also, I just remembered that devops said they're deprecating waffle *switches*, but not flags. I'm not sure what the recommended replacement is, yet, but this should be fine for now.

Reply:
Done.

Oh gosh, I do remember that. If someone mentions it to them, we can remind them they need to update [the OEP on the subject](https://github.com/edx/open-edx-proposals/blob/master/oeps/oep-0017-bp-feature-toggles.rst#use-case-4-ops-graceful-degradation), since it's explicit about recommending the use of switches for most use cases.